pkg/checks/compare.go

```diff
@@ -1,10 +1,5 @@
 package checks
```

Comment:
Is the `DependencyInfo` struct the only thing that is still used? If so I would just move it to `merge.go` and delete this file.

Reply:
That would cause a circular dependency

Reply:
Wait, why? They are in the same package